app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -1533,6 +1545,41 @@ protected void onPause() {
         if (mLocationServiceIssueReceiver != null) {
             unregisterReceiver(mLocationServiceIssueReceiver);
         }
+
+        saveInlineVideoState();
+    }
+
+    private void saveInlineVideoState() {
+        for (int i = 0; i < questionsView.getWidgets().size(); i++) {
+            QuestionWidget q = questionsView.getWidgets().get(i);
+            if (q.findViewById(MediaLayout.INLINE_VIDEO_PANE_ID) != null) {
+                VideoView inlineVideo = (VideoView)q.findViewById(MediaLayout.INLINE_VIDEO_PANE_ID);
+                if (inlineVideo.isPlaying()) {
+                    indexOfWidgetWithVideoPlaying = i;
+                    positionOfVideoProgress = inlineVideo.getCurrentPosition();
+                    return;
+                }
+            }
+        }
+    }
+
+    private void restoreInlineVideoState() {
+        if (indexOfWidgetWithVideoPlaying != -1) {
+            QuestionWidget widgetWithVideoToResume = questionsView.getWidgets().get(indexOfWidgetWithVideoPlaying);
```

Comment:
It will, but I don't think that's relevant...What matters is just that `restoreInlineVideoState` sets the value of `indexOfWidgetWithVideoPlaying` back to -1 (line 1580 below) after restoring. (Not trying to nitpick, just want to make sure I'm understanding your concern and that it's not actually a problem).